api/src/main/java/edu/wpi/first/shuffleboard/api/components/ActionList.java

```diff
@@ -0,0 +1,90 @@
+package edu.wpi.first.shuffleboard.api.components;
+
+import edu.wpi.first.shuffleboard.api.util.FxUtils;
+import edu.wpi.first.shuffleboard.api.util.TypeUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import javafx.scene.Node;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuItem;
+
+public class ActionList {
```

Comment:
What is this class? What does it do? How is it intended to be used? Documentation, please!